tests/test_permissions.py

```diff
@@ -130,20 +124,33 @@ def test_site(request):
         assert permission is False, 'username: "{username}"'.format(
             username=username)
 
-    def test_multiple_user_orgs(self, monkeypatch, settings):
+    @pytest.mark.parametrize('username, allow', [
+        ('regular_user', False),
+        ('staff_user', True),
+        ('super_user', True),
+        ('superstaff_user', True),
+        ('alpha_nonadmin', False),
+        ('alpha_site_admin', True),
+        ('nosite_staff', False),
+        ])
+    def test_multiple_user_orgs(self, monkeypatch, settings, username, allow):
+        """
+        We updated `figures.permissions` so that a user can belong to multiple
+        organizations
+        """
         def test_site(request):
             return self.site
-        username = 'alpha_site_admin'
         request = APIRequestFactory().get('/')
         request.META['HTTP_HOST'] = self.site.domain
         request.user = get_user_model().objects.get(username=username)
         monkeypatch.setattr(django.contrib.sites.shortcuts, 'get_current_site', test_site)
         settings.FEATURES['FIGURES_IS_MULTISITE'] = True
         assert figures.helpers.is_multisite()
         org2 = OrganizationFactory(sites=[self.site])
-        UserOrganizationMappingFactory(user=request.user, organization=org2),
-        with pytest.raises(figures.permissions.MultipleOrgsPerUserNotSupported):
-            figures.permissions.IsSiteAdminUser().has_permission(request, None)
+        UserOrganizationMappingFactory(user=request.user, organization=org2)
+        permission = figures.permissions.IsSiteAdminUser().has_permission(request, None)
+        assert permission is allow, 'User "{username}" should have access'.format(
```

Comment:
```suggestion
        assert permission == allow, 'User "{username}" should have access'.format(
```

I don't think `is` should be used for primitives. There's a reason, but I forgot why :sweat_smile:

Reply:
I'll change it, thanks @OmarIthawi !